CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+* Add stats for Wanderer and Cull relics when using the Relic Stats mod
```

Comment:
Feel free to append your preferred name here

Reply:
Thanks, I edited this entry and put my name in `docs/credits.md` as well.